### PR TITLE
Automate CRDs migration on update

### PR DESCRIPTION
**Pull request by fhielpos**
This PR adds automation to the CRDs migration we need to do on every update.

Here is an example commit:
https://github.com/giantswarm/kyverno-app/pull/212/commits/9686255e9b00925b1dc3e9aa790edaff34a8038b

### CODE REVIEW

File: .github/workflows/sync-from-upstream.yaml [hunk 1]
```diff
@@ -28,8 +28,8 @@ jobs:
           echo "Kyverno last tag found is $KYVERNO_LAST_TAG"
 
           # Setup git config
-          git config user.name github-actions
-          git config user.email github-actions@github.com
+          git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
```
Comment (stone-z) [inline .github/workflows/sync-from-upstream.yaml:31]: Just curious, why?

Also TIL `[]` are actually valid RFC 5322 characters. Gonna go break stuff with that

File: .github/workflows/sync-from-upstream.yaml [hunk 2]
```diff
@@ -44,3 +44,40 @@ jobs:
...
+          # Remove previous CRDs
+          git rm -f *.yaml
+          
+          # Move new CRDs to crd directory
+          cp ../charts/kyverno/templates/crds.yaml .
+
+          # Remove helm content from CRDs
+          APP_VERSION=$(yq '.appVersion' ../charts/kyverno/Chart.yaml)
+          sed -i -e "s/{{.Chart.AppVersion}}/$APP_VERSION/g" crds.yaml
+          sed -i -e "/{{/d" crds.yaml
```
Comment (stone-z) [inline .github/workflows/sync-from-upstream.yaml:68]: Looking forward to the inevitable next Helm templating in CRDs, but this will work for now

**Review by stone-z — APPROVED** [review]
Nonsense comments. Looks good to me :rocket:

File: .github/workflows/sync-from-upstream.yaml (hunk 2 above)
Comment (fhielpos) [reply .github/workflows/sync-from-upstream.yaml:68]: Yeah, this is a pain 🙃

**Pull request merged**